tgui/packages/tgui/interfaces/common/WirePanel/Wire.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file
+ * @copyright 2023
+ * @author glowbold (https://github.com/pgmzeta)
+ * @license MIT
+ */
+
+import { RemoteAccessButton } from ".";
+import { LabeledList } from "../../../components";
+import { WirePanelAction } from "./const";
+import { WireData } from "./type";
+
+interface WireListProps {
+  wires: WireData[];
+  act_wire: (wire_index: number, action: WirePanelAction) => void
+}
+
+export const WireList =(props: WireListProps) => {
+  const { wires, act_wire } = props;
+  return (
+    <LabeledList>
+      {
+        wires.map((wire, i) => (
+          <Wire
+            key={wire.color_name}
+            wire={wire}
+            act_wire={act_wire}
+            dm_index={i + 1}
```

Comment:
This is probably fine, but sanity check on if there's any value in returning the wire color key or whatever instead of its index? Fine if not, I'm not looking at the DM code alongside this right now.